src/DynamoCore/Models/WorkspaceModel.cs

```diff
@@ -786,6 +794,51 @@ internal void IncrementPasteOffset()
             this.currentPasteOffset = (this.currentPasteOffset + PASTE_OFFSET_STEP) % PASTE_OFFSET_MAX;
         }
 
+        internal void SetWorkspaceToState(PresetState state)
```

Comment:
Just call this `ApplyPresetState`.  It should be implicit that we're talking about the `Workspace` here.